src/katgpucbf/fgpu/engine.py

```diff
@@ -315,6 +336,14 @@ def format_complex(value: numbers.Complex) -> str:
     return f"{value.real}{value.imag:+}j"
 
 
+def dig_pwr_dbfs_status(value: float) -> aiokatcp.Sensor.Status:
+    """Compute status for dig-pwr-dbfs sensor.
+
+    TODO: the thresholds are inherited from MeerKAT. Are they what we want?
+    """
+    return aiokatcp.Sensor.Status.NOMINAL if -32.0 <= value <= -22.0 else aiokatcp.Sensor.Status.WARN
```

Comment:
I have a mild recommendation here to put these as constants in `meerkat.py`.

Reply:
I've definitely up for making them constants, but I'm not sure if meerkat.py is the right place. The code outside of meerkat.py is supposed to be instrument-independent, and if it starts depending on meerkat.py then it isn't any more.

Reply:
Hmmm, fair point.
I'd need to think about it a bit more but I'm happy to leave this as it is for now.

Reply:
I've added constants to katgpucbf.fgpu.